### PR TITLE
nordic_nrf: Add linker files

### PR DESCRIPTION
**Pull request by oyvindronningstad**
copied from AN519 with modifications in comments.

Signed-off-by: Øyvind Rønningstad <oyvind.ronningstad@nordicsemi.no>

### CODE REVIEW

**Review by anangl — APPROVED** [review]

**Comment by oyvindronningstad** [comment]
@ioannisg These are mostly copy-pasted, can you take a quick look?

**Review by ioannisg — APPROVED** [review]

**Comment by ioannisg** [comment]
@oyvindronningstad can you fix the commit message to a zephyr-like one?
Also, I don't understand how these linker scripts are different from existing (I compared with AN521). What are the "comments" you are talking about in the commit msg?

**Comment by oyvindronningstad** [comment]
> @oyvindronningstad can you fix the commit message to a zephyr-like one?
> Also, I don't understand how these linker scripts are different from existing (I compared with AN521). What are the "comments" you are talking about in the commit msg?

I had some unpushed changes I had forgotten. Anyways, the differences are only in a couple of comments where "mps2" was mentioned.

File: platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_bl2.ld [hunk 1]
```diff
@@ -0,0 +1,190 @@
+;/*
+; * Copyright (c) 2018-2019 Arm Limited
```
Comment (ioannisg) [inline platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_bl2.ld:2]: @oyvindronningstad can you fix this, it looks a bit weird

File: platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_ns.ld [hunk 2]
```diff
@@ -0,0 +1,188 @@
+;/*
```
Comment (ioannisg) [inline platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_ns.ld:1]: and here

**Comment by oyvindronningstad** [comment]
@ioannisg I added a line to the copyrights. Also, is the commit message ok now?

**Comment by ioannisg** [comment]
> @ioannisg I added a line to the copyrights. Also, is the commit message ok now?

Looks fine, thanks.

**Pull request merged**